src/pages/MyEarnings/ClassicPools/SinglePool/Position.tsx

```diff
@@ -0,0 +1,215 @@
+        }}
+      >
+        {label}
+      </Text>
+
+      {value}
+    </Flex>
+  )
+}
+
+type Props = {
+  chainId: ChainId
+  poolEarning: ClassicPositionEarningWithDetails
+}
+const Position: React.FC<Props> = ({ poolEarning, chainId }) => {
+  const theme = useTheme()
+  const [showEarningChart] = useShowMyEarningChart()
+  const mobileView = useMedia(`(max-width: ${WIDTHS[2]}px)`)
+
+  const myLiquidityBalance =
+    poolEarning.liquidityTokenBalanceIncludingStake !== '0' && poolEarning.pool.totalSupply !== '0'
+      ? formatDollarAmount(
+          (+poolEarning.liquidityTokenBalanceIncludingStake * +poolEarning.pool.reserveUSD) /
+            +poolEarning.pool.totalSupply,
+        )
+      : '--'
+
+  const myShareOfPool = +poolEarning.liquidityTokenBalanceIncludingStake / +poolEarning.pool.totalSupply
+
+  const pooledToken0 = +poolEarning.pool.reserve0 * myShareOfPool
+  const pooledToken1 = +poolEarning.pool.reserve1 * myShareOfPool
+
+  const token0 = useMemo(
+    () =>
+      new Token(
+        chainId,
+        poolEarning.pool.token0.id,
+        +poolEarning.pool.token0.decimals,
+        poolEarning.pool.token0.symbol,
+        poolEarning.pool.token0.name,
+      ),
+    [chainId, poolEarning],
+  )
+
+  const token1 = useMemo(
+    () =>
+      new Token(
+        chainId,
+        poolEarning.pool.token1.id,
+        +poolEarning.pool.token1.decimals,
+        poolEarning.pool.token1.symbol,
+        poolEarning.pool.token1.name,
+      ),
+    [chainId, poolEarning],
+  )
+
+  const liquidityStaked = +poolEarning.liquidityTokenBalanceIncludingStake - +poolEarning.liquidityTokenBalance
+  const myStakedBalance =
+    liquidityStaked !== 0
+      ? formatDollarAmount((liquidityStaked * +poolEarning.pool.reserveUSD) / +poolEarning.pool.totalSupply)
+      : '--'
+
+  const stakedShare = liquidityStaked / +poolEarning.pool.totalSupply
+
+  const stakedToken0 = +poolEarning.pool.reserve0 * stakedShare
+  const stakedToken1 = +poolEarning.pool.reserve1 * stakedShare
+
+  return (
+    <Flex
+      flexDirection="column"
+      backgroundColor={mobileView ? 'transparent' : theme.background}
+      padding={mobileView ? '1rem 0' : '1rem'}
+      margin={mobileView ? 0 : '0 0.75rem 0.75rem'}
+      sx={{ borderRadius: '1rem', gap: '1rem' }}
+    >
+      <Text fontSize={16} fontWeight="500">
+        <Trans>My Liquidity Positions</Trans>
+      </Text>
+
+      {showEarningChart && <PoolEarningsSection historicalEarning={poolEarning.historicalEarning} chainId={chainId} />}
+
+      <Box
+        sx={{
+          width: '100%',
+          display: 'flex',
+
+          justifyContent: 'space-between',
+
+          background: theme.buttonBlack,
+          borderRadius: '20px',
+          padding: '24px',
+          gap: '8px',
+
+          ...(mobileView && {
+            display: 'grid',
+            gridTemplateColumns: '1fr 1fr',
+            padding: '0',
+            gap: '0.75rem',
+            '> *:nth-child(2n)': {
```

Comment:
Amazing, does it work?

Reply:
ofc, works fine
<img width="642" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/11427213/dca9d8c0-8f31-42dd-9a3d-9bf4a723cdcb">